src/main/java/org/apache/sling/api/resource/ResourceResolver.java

```diff
@@ -842,4 +848,20 @@ Resource copy(final String srcAbsPath,
      */
     Resource move(final String srcAbsPath,
               final String destAbsPath) throws PersistenceException;
+
+    /**
+     * Returns a map to store temporary objects.
+     *
+     * This map is suited to store objects which share the same lifecycle as the
+     * resource resolver. The resource resolver itself does not use this map.
+     *
+     * The resource resolver will clear the map during {@link #close()}, so afterwards the map is empty.
+     * If a stored value implements the {@link #Closeable} interface, the ResourceResolver will invoke the
+     * <code>close()</code> of the value before clearing the map.
+     *
+     * @returns the property map
+     * @see #close()
+     * @since 2.13.0
```

Comment:
Let us add the bundle version as well. Consumers need to know rather that version!

Reply:
```suggestion
     * @since 2.13.0 (Sling API Bundle 2.24.0)
```